### PR TITLE
[Menu] E2E test for Menu Android

### PR DESCRIPTION
**Pull request by rohanprasadofficial**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [x] android

### Description of changes

Seems there is local failure for e2e runs , testing it on the pipeline once.

### Pull request checklist

This PR has considered (when applicable):
- [x] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: apps/E2E/src/Menu/specs/Menu.spec.android.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import MenuPageObject from '../pages/MenuPageObject';
+
+// Before testing begins, allow up to 60 seconds for app to open
+describe('Menu Testing Initialization', () => {
+  it('Wait for app load', async () => {
+    await MenuPageObject.waitForInitialPageToDisplay();
```
Comment (ankraj12) [inline apps/E2E/src/Menu/specs/Menu.spec.android.ts:6]: You can consider adding more functionality testing as part of E2E given Menu is an interactive component..

**Review by ankraj12 — APPROVED** [review]
Leaving a note for for review

**Comment by rohanprasadofficial** [comment]
Will be adding them along with bug bash fixes as there will be some changes in position logic.

File: apps/E2E/src/Menu/specs/Menu.spec.android.ts (hunk 1 above)
Comment (rohanprasadofficial) [reply apps/E2E/src/Menu/specs/Menu.spec.android.ts:6]: @ankraj12  , Will be adding them along with bug bash fixes as there will be some changes in position logic.

**Pull request merged**